frontend/projects/abtesting/src/app/features/dashboard/home/components/experiment-post-condition/experiment-post-condition.component.ts

```diff
@@ -40,7 +40,7 @@ export class ExperimentPostConditionComponent implements OnInit, OnChanges {
         this.newExperimentData = experimentData;
         this.experimentInfo = experimentData;
       });
-    } else if (this.data) {
+    } else if (this.data && this.data.experiment !== undefined) {
```

Comment:
@ppratikcr7 You can directly write it like `this.data && this.data.experiment`. Please update the code here